tests/antithesis/test-template/go-delete-keys/serial_driver_delete_keys.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
Please build the go binary into the client docker image. Like in https://github.com/antithesishq/etcd-test-composer/blob/ec57429a8cbc9dc39fb76b1b1a8250712ee4d283/test-template/Dockerfile.client#L57-L72